NAMESPACE

```diff
@@ -387,6 +387,7 @@ export(layer_scales)
 export(layer_sf)
 export(lims)
 export(map_data)
+export(mapped)
```

Comment:
I think `mapped()` (or the functionality it provides) will be used more than stage, so IMO it makes sense to provide a `stat()` parallel

Reply:
I agree with Thomas. I'd like to have access to something like `mapped()`. `stage()` is sufficiently complicated that in most cases I would probably not bother using it. (Though it's good to have the option.)

Reply:
If we're going to have a collection of these functions, I think we need to think about the name a bit more — I think a common prefix would be appropriate.

Reply:
I think the ship has sailed for that since we already have `stat()`... Open for something other than `mapped()` but I don't think we can meaningfully introduce prefixes at this point.

Reply:
In that case, I think the second argument to `stage()` can't be `geom`, since if we did add an explicit function for that case, we couldn't call it `geom()`.

I also wouldn't object to `stage_mapped()`, `stage_stat()`, etc. Although then I guess we'd have to soft deprecate `stage()`?

Reply:
The current `stage()` function needs some rethinking. Currently the `stat` argument corresponds to default evaluation, and the `geom` argument corresponds to `stat()` evaluation which is counterintuitive. The problem is that `stat()` is intended to denote the mapping happens after stats have been computed whereas `stage()` refers to mapping of the next stage. Maybe the signature should be `stage(start, after_stat, after_scale)` with corresponding `after_stat()` and `after_scale()` functions (soft-deprecating `stat()`)

Reply:
I rather like that idae